src/marketing/search-engine-robots.md

```diff
@@ -2,7 +2,7 @@
 title: Search Engine Robots
 ---
 
-The Magento configuration includes settings to generate and manage instructions for web crawlers and bots that index your site. The instructions are saved in a file called `robots.txt` that resides in the root of your Magento installation. The instructions are directives that are recognized and followed by most search engines.
+The Magento configuration includes settings to generate and manage instructions for web crawlers and bots that index your site. If the request for `robots.txt` reaches Magento (rather than a physical file) [it is dynamically routed to the robots controller](https://github.com/magento/magento2/blob/2.3.5-p2/app/code/Magento/Robots/Controller/Router.php#L59). The instructions are directives that are recognized and followed by most search engines.
```

Comment:
We don't link directly to code from the user guide, but this is a good reference link for the PR.

```suggestion
The Magento configuration includes settings to generate and manage instructions for web crawlers and bots that index your site. If the request for `robots.txt` reaches Magento (rather than a physical file) it is dynamically routed to the robots controller. The instructions are directives that are recognized and followed by most search engines.
```